index.html

```diff
@@ -1,173 +1,218 @@
+    
+                <ul class="languages">
+                    <li class="language">
+                        <div class="languages-logo"><img src="image-source/Ellipse 1.png" alt="javascript"></div>
+                        <p>javascript</p>
+                    </li>
+                    <li class="language">
+                        <div class="languages-logo"><img src="image-source/Ellipse 2.png" alt="javascript"></div>
+                        <p>HTML</p>
+                    </li>
+                    <li class="language">
+                        <div class="languages-logo"><img src="image-source/Ellipse 3.png" alt="javascript"></div>
+                        <p>CSS</p>
+                    </li>
+                </ul>
+    
+                <ul class="framework">
+                    <li>Frameworks</li>
+                    <div class="vector"><img src="image-source/Icon - Chevron - Right - Larger.png" alt=""></div>
                 </ul>
-            </section>
-            </section>
-        </div>    
-        </div>
-    </body>
+    
+                <ul class="framework">
+                    <li>Skills</li>
+                    <div class="vector"><img src="image-source/Icon - Chevron - Right - Larger.png" alt=""></div>
+                </ul>
+            </div>
+        </section>
+
+        <section id="Contact" class="contact-section">
+            <img src="image-source/Ellipse 12.png" class="ellipse-12 el" alt="Ellipse 12">
+            <img src="image-source/Ellipse 33.png" class="ellipse-33 el" alt="Ellipse 33">
+            <img src="image-source/Rectangle 4.png" class="rectangle-4 el" alt="Rectangle 4">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+            <form action="https://formspree.io/f/mknagznd" method="post">
+                <fieldset class="contact-form">
+                    <label for="username">
+                        <input type="text" class="input-contact" required name="username" maxlength="30" placeholder="Username">
+                    </label>
+                    <label for="email">
+                        <input type="email" class="input-contact" required name="email" placeholder="name@domain.com">
+                    </label>
+                    <label for="message">
+                        <textarea name="message" class="message-input" required maxlength="500" placeholder="Write your message here"></textarea>
+                    </label>
+                    <div class="button">
+                      <button type="submit" class="btn last-btn">Get It Touch</button>
+                    </div>
+                </fieldset>
+                <img src="image-source/Ellipse 5.png" class="ellipse-5 el" alt="ellipse-5">
+                <img src="image-source/Rectangle 7.png" class="rectangle-7 el" alt="rectangle-7">
+            </form>
```

Comment:
Please consider adding `border-radius: 2px;` on all the input fields including text-area. It will make it more alike the Template.

Reply:
Thanks for the review @ABDULALI3468